### DISCUSSION TITLE
Moveable Scoreboard in Skin Editor

### DISCUSSION THREAD

**Discussion by MalfunctionCode** [ppy/osu#30694] [Ideas] [answered]
Hey there, sorry if this has been discussed already but I couldn't find anyone mentioning this Issue. 

On all Legacy Skin the Scoreboard is positioned in the same spot in the top left, which overlaps over most healthbars:
![osu!_ZqoHmVGoT4](https://github.com/user-attachments/assets/21acf36e-931c-49dc-9cd8-97d78fe4085c)

From what I've read, the Scoreboard should dynamically Position itself based on the Healthbar location? Atleast that's what I think is happening on the argon skins. I guess this dynamic positioning dosen't include the legacy healthbars, but in general an option to move/skin the scoreboard would be nice feature to have.

Thanks in Advance!

**Comment by Joehuu** [comment, marked as answer]
> From what I've read, the Scoreboard should dynamically Position itself based on the Healthbar location?

It can't because of this:
https://github.com/ppy/osu/blob/bf2e0ed005662edbe72ba941d5a34ad364dd3ee7/osu.Game/Screens/Play/HUDOverlay.cs#L299-L301

But yeah, it will eventually be skinnable so you can move it anywhere.
